src/components/DropDownMenu.js

```diff
@@ -1,10 +1,57 @@
 import React from 'react';
 
 class DropDownMenu extends React.Component {
+    constructor() {
+        super();
+
+        this.state = {
+            showCheese: false,
+        };
+
+        this.showCheese = this.showCheese.bind(this);
+
+    }
+
+    showCheese(event) {
+        event.preventDefault();
+
+        this.setState({ showCheese: true }, () => {
```

Comment:
Could we change true to !showCheese ? Then it should toggle between the two. We might not need a close cheese function then.